src/test/java/com/cloudbees/jenkins/support/actions/SupportAbstractItemActionTest.java

```diff
@@ -77,11 +75,10 @@ public void generateFreestyleBundleDefaultsAndCheckContent() throws Exception {
     }
 
     /*
-     * Integration test that simulates the user action of clicking the button to generate the bundle from a Folder.
+     * Integration test that simulates the user action of clicking the button to generate the bundle from a Pipeline job.
```

Comment:
I realised this changed from Folder to Pipeline? Was it actually wrong before?

Reply:
To original comment looks like a copy'n' paste error from line 32.
This test is for a Pipeline job not a Folder, see its test method name.